### ISSUE TITLE
Symfony 6 support

### ISSUE THREAD

**Issue by codedge** [Codeception/module-symfony#152]
Currently it is not possible to use the module with Symfony 6 as the reqs in `composer.json` only a Symfony 4 and 5.
Any plans to support Symfony 6?

**Comment by TavoNiievez** [comment]
Hi @codedge ,

So far there is no `codeception/codeception` release tagged with Symfony 6 support, that's our blocker right now. 

Support has already been implemented in `5.0` branch but, `5.0.0` in turn is blocked by PHPUnit `10.0.0` which will *probably* be released next month.

**Comment by codedge** [comment]
According to this issue https://github.com/Codeception/Codeception/issues/6294 I hope the `5.0` release is going to happen in the next days as PHPUnit seem to be out of scope for `5.0`.

**Comment by TavoNiievez** [comment]
The last comment in that thread is a month old, we decided to keep waiting for PHPUnit 10 after that...

**Comment by semaz** [comment]
Hello. 
To support php 8 and symfony 6, two lines in the file `src/Codeception/Module/Symfony/SessionAssertionsTrait.php` need to be corrected:
1. Replace the content of the "getCurrentSession()" method with `return $this->grabService('session.factory')->createSession();`
2. In the `amLoggedInAs` function, you need to remove the second argument from the `UsernamePasswordToken` class so line 43 must looks like this `$token = new UsernamePasswordToken($user, $firewallName, $user->getRoles());`

So far, I've made these changes to my helper class. But I hope that you will add them to the module code.

**Comment by TavoNiievez** [comment]
@semaz Cool.
The tests in [`symfony-module-tests` ](https://github.com/Codeception/symfony-module-tests/blob/c9b8692083a3afd14828e9c1b4006e0b6308d852/tests/Functional/SessionCest.php#L70)also need to be adapted, for example making the change to `getCurrentSession()` prints:
```
 Step  See in session "_security_main"
 Fail  No session attribute with name '_security_main'
```
it may be that this session variable has been renamed,
if you have time you can try to send PRs to add this support, I will be very attentive to review it, otherwise I'll do both myself.

**Comment by semaz** [comment]
#157 Hi @TavoNiievez. I created first PR in my life) 
Please review it.

**Comment by TavoNiievez** [comment]
It seems to be a special occasion, I'm glad that Codeception is the project that receives your first PR.
**Friendly reminder**: we have some [`contribution guides`](https://github.com/Codeception/module-symfony/blob/main/CONTRIBUTING.md) in case you want to learn more about our workflow and how `module-symfony` uses [`symfony-module-tests`](https://github.com/codeception/symfony-module-tests) to perform tests.
